### PR TITLE
feat(logstash): improve logstash provisioning

### PR DESCRIPTION
**Pull request by serge1peshcoff**
1) passing env vars to logstash from .env
2) using this vars in logstash.yml
3) added docker-compose for dev setup so it'd work
4) removed output block that wasn't used

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -95,3 +95,7 @@ PATH_OMS_SURVEY=oms-survey/docker/
...
+
+ELASTIC_HOST=https://example.com:9243
```
Comment (linuxbandit) [inline .env.example]: already on line 36-38

File: oms-monitor/docker/docker-compose.yml [hunk 2]
```diff
@@ -94,6 +94,9 @@ services:
       - "9600:9600"
     environment:
       LS_JAVA_OPTS: "-Xmx256m -Xms256m"
+      ELASTIC_HOST: "${ELASTIC_HOST}"
+      ELASTIC_USER: "${ELASTIC_USER}"
+      ELASTIC_PASSWORD: "${ELASTIC_PASSWORD}"
```
Comment (linuxbandit) [inline oms-monitor/docker/docker-compose.yml:99]: really? does it work this way?

**Review by linuxbandit — COMMENTED** [review]
what is different from the other one you closed?

File: oms-monitor/docker/docker-compose.yml (hunk 2 above)
Comment (serge1peshcoff) [reply oms-monitor/docker/docker-compose.yml:99]: yep, this is on b1 right now and it works

File: .env.example (hunk 1 above)
Comment (serge1peshcoff) [reply .env.example]: didn't notice, will check

**Comment by serge1peshcoff** [comment]
@linuxbandit the difference is that in the previous version it used cloud.id and cloud.yml, doesn't work this way apparently. This works.

File: .env.example (hunk 1 above)
Comment (serge1peshcoff) [reply .env.example]: fixed

File: oms-monitor/docker/logstash/pipeline/logstash.conf [hunk 3]
```diff
@@ -118,17 +118,9 @@ filter {
...
-    }
-  } else {
-    elasticsearch {
-      hosts => "${ELASTIC_INSTANCE}"
-      user => "${ELASTIC_USER}"
-      password => "${ELASTIC_PASSWORD}"
-    }
+  elasticsearch {
+    hosts => "${ELASTIC_HOST}"
+    user => "${ELASTIC_USER}"
```
Comment (linuxbandit) [inline oms-monitor/docker/logstash/pipeline/logstash.conf:123]: so this works without even a `envsubst` ?? wow

Comment (serge1peshcoff) [reply oms-monitor/docker/logstash/pipeline/logstash.conf:123]: yep: https://www.elastic.co/guide/en/logstash/current/environment-variables.html

File: oms-monitor/docker/docker-compose.yml [hunk 4]
```diff
@@ -94,6 +94,9 @@ services:
       - "9600:9600"
     environment:
       LS_JAVA_OPTS: "-Xmx256m -Xms256m"
+      ELASTIC_HOST: "${ELASTIC_HOST}"
+      ELASTIC_USER: "${ELASTIC_USER}"
```
Comment (linuxbandit) [inline oms-monitor/docker/docker-compose.yml:98]: it's called ID above

**Review by linuxbandit — CHANGES_REQUESTED** [review]

File: .env.example [hunk 5]
```diff
@@ -33,9 +33,9 @@ SENDGRID_SMTP_HOST=smtp.sendgrid.net
 SENDGRID_USER=
 SENDGRID_PW=5ecr3t
 
-ELASTIC_INSTANCE=
-ELASTIC_USER=
-ELASTIC_PASSWORD=
+ELASTIC_HOST=https://example.com:9243
+ELASTIC_ID=5ecr3t
```
Comment (linuxbandit) [inline .env.example]: it's called ELASTIC_USER below

**Review by linuxbandit — CHANGES_REQUESTED** [review]

File: .env.example (hunk 5 above)
Comment (serge1peshcoff) [reply .env.example]: Weird, I thought I've updated it. Will fix in 30'

Comment (serge1peshcoff) [reply .env.example]: fixed

**Review by linuxbandit — APPROVED** [review]

**Pull request merged**